pom.xml

```diff
@@ -425,6 +425,7 @@
     <surefire.version>2.22.1</surefire.version>
 
     <surefire-forkcount>8</surefire-forkcount>
+    <redirectTestOutputToFile>true</redirectTestOutputToFile>
```

Comment:
this part of the change is not strictly needed, I will revert once I fixed test flakyness